UI_UX/Frontend/src/components/AccountInfo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+import { Address } from './Address';
+
+export const AccountInfo = ({
+  loadingBalance,
+  loadingAccountInfo,
+  myBitBalance,
+  ethBalance,
+  address
+}) => {
+  let loadingIcon = undefined;
+  if (loadingBalance)
+    loadingIcon = (
+      <Loading
+        className="AppHeader_balance_loading"
+        small
+        withOverlay={false}
+      />
+    );
+
+  return (
+    <div className="AppHeader_account_info">
+      <div className="AppHeader_balance">
+        <p className="bold">Balance</p>
+        {loadingBalance ? (
+          loadingIcon
+        ) : (
+          <span>
+            {myBitBalance} <span className="bold">MYB</span> &nbsp; {ethBalance}{' '}

```

Comment:
No unnecessary semantic classes. If it needs to be bold, use the `<b>` tag instead. Furthermore, avoid using `&nbsp;`, styling is done in css, and in this case you are adding spacing with HTML code. Go for a `padding-left: X` instead.

What's the purpose of the `{' '}` tag?

Reply:
fixed. Unsure about the tag and how it ended up there, hm.